core/src/actions/properties.ts

```diff
@@ -48,17 +54,19 @@ export class PropertiesList extends AuthenticatedAction {
       where["unique"] = true;
     }
 
+    const include: Includeable[] = [
+      {
+        model: Source,
+        where: { state: ["draft", "ready"] },
+      },
+    ];
+
     const properties = (
       await Property.scope(null).findAll({
-        include: [
-          {
-            model: Source,
-            where: { state: ["draft", "ready"] },
-          },
-        ],
-        limit: params.limit,
-        offset: params.offset,
-        order: params.order,
+        include,
+        limit: Number(params.limit),
+        offset: Number(params.offset),
+        order: params.order as unknown as Order,
```

Comment:
I originally added in the `invalid` checking functionality here, but ended up not needing it. These changes don't really hurt anything I feel, but can revert.

Reply:
I think this was left over from before I understood the formatter work. Removing